docs/src/assets/scss/docs.scss

```diff
@@ -157,3 +157,34 @@ pre[class*="language-"] {
         opacity: 1;
     }
 }
+
+.scroll-up-btn {
+    width: 50px;
+    height: 50px;
+    display: none;
+    position: fixed;
+    right: 50px;
+    bottom: 35px;
+    font-size: 1.5rem;
+    border-radius: 50%;
+    color: #fff;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(75, 50, 195, 0.9); // rgba of #4b32c3
```

Comment:
lets use css variables for colors `--color-primary-800` is the variable you are looking for

Reply:
the rgba form of these theme color is used to make them transparent a bit. didn't use opacity because it is also making the arrow transparent as well.

Reply:
I thought that It would be good to keep this button transparent so it doesn't feel like that it is blocking something in the page.

Reply:
Most of the time the user should be able to scroll a little more to see content below the button? I think its fine to not include opacity. adding opacity will just make it harder to see content below.

Reply:
I think not include opacity is better, the user can see a little through the button but if it is not completely clear it will only cause misleading.

Reply:
Yes, opacity is removed now completely.